assets/js/googlesitekit/datastore/site/consent-mode.js

```diff
@@ -195,6 +198,18 @@ const baseSelectors = {
 			return undefined;
 		}
 
+		const googleTagContainerDestinationIDs =
+			getGoogleTagContainerDestinationIDs();
+
+		if (
+			Array.isArray( googleTagContainerDestinationIDs ) &&
```

Comment:
Hmm. It looks like this selector will implicitly return `undefined` if `googleTagContainerDestinationIDs` is itself `undefined`, because `adsConversionID` and `adsLinked` will also be `undefined` in this scenario.

However, I think it would be worth being more explicit for the sake of clarity, and including a check for `googleTagContainerDestinationIDs` on line 197:

https://github.com/google/site-kit-wp/blob/8e17aab5045bc9a4596b60d12112b33c38dcbe87/assets/js/googlesitekit/datastore/site/consent-mode.js#L197-L199

Reply:
Updated, thank you!